.github/workflows/build.yml

```diff
@@ -40,6 +40,12 @@ jobs:
           - m32: ${{ github.event_name == 'pull_request' && 1              || 'do-not-exclude' }}
           # Build -m32 only on ubuntu-20.04
           - {os: ubuntu-18.04, m32: 1}
+        include:
+          # Build GCC 10 on ubuntu-20.04
+          - os: ubuntu-20.04
+            compiler: { cc: gcc-10, cxx: g++-10 }
+            m32: 0
+
```

Comment:
I think the GCC10 enablement for CI testing can be a pull against master.  (But not the config changes for coroutine use).

Reply:
https://github.com/verilator/verilator/pull/3432